app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
```

Comment:
- [ ] For your login, Please make sure a user can log in with a combination of email and password. Your application asks to confirm my email before signing in (which is good), but It won't send a confirmation email.

https://user-images.githubusercontent.com/69077061/223698451-82ba7e8a-77dd-40ee-875a-ca355c4738f1.mp4